client/tsconfig.json

```diff
@@ -19,16 +18,12 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
+    "strictPropertyInitialization": false,
     "plugins": [
       {
         "name": "next"
       }
     ],
-    "paths": {
```

Comment:
@alexeh do we really need to get rid of this? I prefer `@/{any_folder_under_src}` than `@/client/src/{any_folder_under_src}`, I find the second one a little redundant and longer. Also, in my mind at least, the _client_ is not self-aware of being a _client_, that's how you call the application when it is used from outside.